modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/search_bar.js

```diff
@@ -26,7 +26,7 @@ AUI.add(
 				'.search-bar-empty-search-input'
 			);
 
-			if (emptySearchInput.val() === 'true') {
+			if (emptySearchInput && emptySearchInput.val() === 'true') {
```

Comment:
Why not just do

`instance.emptySearchEnabled = emptySearchInput && emptySearchInput.val() === 'true'` instead of all this `if/else` stuff?

Reply:
I'll add it in! Thanks!